### PR TITLE
Update deps, fix closure compilation

### PR DESCRIPTION
**Pull request by RaananW**
Dependency update
Also fixing a compilation issue we have with closure-compiler (optional should be marked with [] in doc)

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15427/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15427/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15427/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15427/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15427/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15427/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15427/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15427/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15427/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15427/merge/testResults/webgpuplaywright/index.html

File: packages/dev/core/src/Actions/condition.ts [hunk 1]
```diff
@@ -135,7 +135,7 @@ export class ValueCondition extends Condition {
         public propertyPath: string,
         /** the value compared by the conditional operator against the current value of the property */
         public value: any,
-        /** the conditional operator, default ValueCondition.IsEqual */
+        /** [number] the conditional operator, default ValueCondition.IsEqual */
```
Comment (ryantrem) [inline packages/dev/core/src/Actions/condition.ts:138]: Just curious, who is using the closure compiler? How are these issues being found? Also curious if we want to make sure our code works with closure, are there ESLint plugins that can help with this?

Comment (RaananW) [reply packages/dev/core/src/Actions/condition.ts:138]: answered offline :-)

Comment (noname0310) [reply packages/dev/core/src/Actions/condition.ts:138]: I'm personally curious, too

Comment (RaananW) [reply packages/dev/core/src/Actions/condition.ts:138]: I can answer the how part 😀
We have a CI pipeline running daily that compiles core, loaders, and serializers packages using closure. If it fails, we get notified.

**Pull request merged**